src/sentry/lang/dart/apps.py

```diff
@@ -0,0 +1,12 @@
+from django.apps import AppConfig
+
+
+class Config(AppConfig):
+    name = "sentry.lang.dart"
+
+    def ready(self):
+        from sentry.plugins.base import register
+
+        from .plugin import DartPlugin
+
+        register(DartPlugin)
```

Comment:
This currently isn't being used, can I avoid registering the plugin by just removing this file? Commenting it out? I've disabled the plugin in `src/sentry/lang/dart/plugin.py`